package.json

```diff
@@ -51,6 +51,7 @@
     "react": "*"
   },
   "jest": {
+    "testURL": "http://localhost",
```

Comment:
I was not able to run the test suite locally without adding this line - I am curious how the test suite was running on your machine previously, and if this is necessary.

Reply:
i have no idea why a test URL would be necessary at all. are you using `npm test`, or `jest` globally?

(this shouldn't be part of this PR regardless)